### ISSUE TITLE
[BUG] Wrong message order makes app incompatible with screen readers (for visually impaired)

### ISSUE THREAD

**Issue by grzebo** [RocketChat/Rocket.Chat.Android#2533]
## Describe the bug
Android phones come with "TalkBack", a built-in screen reader, which is an accessibility app, that reads the content of the screen to visually impaired people. Rocket.chat app is incompatible with TalkBack, because when a user selects more than one message in Rocket.chat with TalkBack, they are read in reverse order (last message is read first). This causes Rocket.chat to be impractical to visually impaired people, since they need to select each message separately if they want to hear the conversation in correct order.

## To Reproduce
1. Enable TalkBack on your Android device.
2. Open Rocket.chat, go to any channel.
3. Using a TalkBack function make a selection that contains more than one message.
4. TalkBack reads messages in reverse order: first the message at the bottom of the screen (latest), then the message above it (earlier one), then one above it (even earlier one) and so on.

## Expected behavior
TalkBack should read messages in the correct chronological order (top to bottom). This is a problem with Rocket.Chat, as all other chat apps work correctly with TalkBack (FB Messenger, Slack, SMS apps, Signal etc.)

## Screenshots
Correct behavior in Slack:
User selects some messages:
![bug1](https://user-images.githubusercontent.com/47195640/73798081-188f0b80-47b2-11ea-9b1d-6b09faedd915.png)
TalkBack starts reading from the top one:
![bug2](https://user-images.githubusercontent.com/47195640/73798087-1cbb2900-47b2-11ea-9cae-f4658d54e4dd.png)

Wrong behavior in Rocket.chat:
User selects some messages:
![bug4](https://user-images.githubusercontent.com/47195640/73798095-22b10a00-47b2-11ea-8b6f-caf4a88f4a8c.png)
TalkBack starts reading the lowest message first, and then reads the top one (reverse order):
![bug3](https://user-images.githubusercontent.com/47195640/73798091-204eb000-47b2-11ea-9075-d0b9f55e674a.png)

## Devices and Versions
Bus exists in latest stable and experimental version of the app, independent of device version.

**Comment by diegolmello** [comment]
Thanks for letting us know!
I'm transferring this issue to the new repo.
We're going to take a look at it soon.
